javascript/utils/idp.js

```diff
@@ -29,6 +29,14 @@ firebaseui.auth.idp.isSupportedProvider = function(providerId) {
 
 
 /**
+ * The provider ID constant for the email auth provider
+ *
+ * @const {string}
+ */
+firebaseui.auth.idp.EMAIL_PROVIDER_ID = 'password';
```

Comment:
Not a good idea to hard code this. It is available from firebase.auth.EmailAuthProvider.PROVIDER_ID